src/obfl-specification.html

```diff
@@ -1281,7 +1283,7 @@ <h4 id="L1438" class="include">br</h4>
 <div class="sidebar markup">
 <dl>
   <dt>Usage</dt>
-    <dd>before, after, toc-entry, block, item</dd>
+    <dd>before, after, toc-entry, block, toc-block, item</dd>
```

Comment:
As ```toc-block``` is used for nesting and text is not allowed then ```br``` is not required or needed.